core/Enclave/enclave_hsm.cpp

```diff
@@ -458,27 +554,103 @@ sgx_status_t enclave_rsa_encrypt(const uint8_t *cmk_blob,
                                  uint8_t keyspec,
                                  uint8_t padding_mode)
```

Comment:
are keyspec and pading_mode still required for encryption/decryption?
I assume the information should be recorded in cmk_blob.
@syan10